lib/transcoder.ex

```diff
@@ -69,16 +69,31 @@ defmodule Membrane.Transcoder do
                 and is supposed to return the desired output stream format or its module.
                 """
               ],
-              force_transcoding?: [
-                spec: boolean() | (stream_format() -> boolean()),
-                default: false,
+              transcoding_policy: [
+                spec:
+                  :always
+                  | :if_needed
+                  | :never
+                  | (stream_format() -> :always | :if_needed | :never),
+                default: :if_needed,
                 description: """
-                If set to `true`, the input media stream will be decoded and encoded, even
-                if the input stream format and the output stream format are the same type.
+                Specifies, when transcoding should be appliead.
 
                 Can be either:
-                * a boolean,
-                * a function that receives the input stream format and returns a boolean.
+                * an atom: `:always`, `:if_needed` (default) or `:never`,
+                * a function that receives the input stream format and returns an atom.
```

Comment:
```suggestion
                * a function that receives the input stream format and returns the atom.
```
or perhaps it would be even better if you listed all the atoms once again